test/drop_type_declaration/thetest.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
```

Comment:
Could we call this file something else and not thetest.proto as well?
How about drop.proto